spec/lang/step/terminators.md

```diff
@@ -90,8 +90,9 @@ fn check_abi_compatibility(
          Type::Tuple { sized_fields: callee_fields, sized_head_layout: callee_head_layout, unsized_field: callee_unsized_field }) => {
             let (caller_size, caller_align) = caller_head_layout.head_size_and_align();
             let (callee_size, callee_align) = callee_head_layout.head_size_and_align();
-            caller_unsized_field.is_none() &&
-            callee_unsized_field.is_none() &&
+            // FIXME: Matching on `unsized_field: None` would be nicer, but specr currently does not support that.
```

Comment:
I don't agree with this, having a single block handling al `Tuple` cases seems quite nice to me.

Reply:
Fair, I was searching for a place where it would be nicer, but if we always want to match only once on tuples, this comment doesn't have a good place anymore.